src/Microsoft.TestPlatform.Client/TestPlatform.cs

```diff
@@ -76,6 +82,8 @@ public ITestRunRequest CreateTestRunRequest(TestRunCriteria testRunCriteria)
                 throw new ArgumentNullException("testRunCriteria");
             }
 
+            UpdateTestAdapterPaths(testRunCriteria.TestRunSettings);
+
```

Comment:
I would like to take a deeper look at the datacollector scenarios. but right now TPv2 does not support DataCollectors.